<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetConfVelocidad</class>
 <widget class="QWidget" name="WidgetConfVelocidad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSlider" name="slHorizontal">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
  </widget>
  <widget class="QSlider" name="slVertical">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>16</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::TicksBothSides</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbColisionar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>colisiona</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
